rotkehlchen/chain/ethereum/modules/uniswap/uniswap.py

```diff
@@ -232,10 +232,10 @@ def _get_balances_graph(
             result_data = result['liquidityPositions']
 
             for lp in result_data:
-                user_address = to_checksum_address(lp['user']['id'])
+                user_address = deserialize_ethereum_address(lp['user']['id'])
```

Comment:
for both of these you will now need to handle a potential `DeserializationError`, log it and `continue` in the loop